### PR TITLE
itemsExpand still not work, fix again

### PR DESCRIPTION
**Pull request by yestaro**
fix itemsExpand and add initStyleToLayout in set strand

### CODE REVIEW

**Comment by carlosrovira** [comment]
Hi Taro,

I want to review your change but I need an example where the actual solution fails. I need to understand where is the fail before looking at the change to know what I'm reviewing. Can you provide a simple test case? Thanks for this PR!! :)

File: frameworks/projects/Jewel/src/main/royale/org/apache/royale/jewel/beads/layouts/StyledLayoutBase.as [hunk 1]
```diff
@@ -77,6 +79,58 @@ package org.apache.royale.jewel.beads.layouts
...
+			/// Names this function - initStyleToLayout
+			///-----------------------------------------
+			var cssValue:* = ValuesManager.valuesImpl.getValue(component, cssProperty);
+			if (cssValue !== undefined)
+			{
+				switch(cssProperty)
+				{
+					case "itemsExpand":
+						if(!itemsExpandInitialized)
+							itemsExpand = Boolean(cssValue);
```
[the remark this answers is not in the document]
Comment (yestaro) [reply frameworks/projects/Jewel/src/main/royale/org/apache/royale/jewel/beads/layouts/StyledLayoutBase.as]: Yes, you are right. I didn't notice it.

**Comment by carlosrovira** [comment]
Hi Yu,
your first code PR is just merged! Thanks for your help!, hoping to see more of this in the near future! :))

**Comment by piotrzarzycki21** [comment]
That's actually the third one :) First two were in Royale-doc. Keep up the good work and you will be rewarded! :)

**Comment by carlosrovira** [comment]
Right! for that reason I said "code PR" since the other ones was for documentation ;-)

**Pull request merged**